package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "eslint-config",
```

Comment:
Should we call it `@pagopa/eslint-config`?
```suggestion
  "name": "@pagopa/eslint-config",
```

Reply:
You are right, is needed the scope into the package name.